internal/service/common/utils/utils.go

```diff
@@ -21,6 +21,16 @@ func (r DBTag) Columns() []any {
 	return columns
 }
 
```

Comment:
I don't see all this being used anywhere. Let's keep this pr clean and introduce `CompareObjects` later when is used

Reply:
I did this intentionally to reduce the size of the upcoming PRs.  I wanted to avoid one huge PR with everything in it since it makes it difficult to review.

Reply:
I understand that and I'm not trying to be picky. It is just a good practice. These parts are not too big. Let's introduce them later.